styles/banner.css

```diff
@@ -0,0 +1,40 @@
+.Cart {
+  font-family: sans-serif;
+}
+
+p {
+  margin: 0;
+}
+
+p:not(:last-of-type) {
+  margin-bottom: 0.5rem;
+}
+
+.banner {
+  display: flex;
+  padding: 1rem;
+}
+
+.banner_container {
+  width: auto;
+  font-family: sans-serif;
+  display: inline-block;
+  margin-right: 30px;
```

Comment:
I don't think we need this margin as we already have a padding in the Banner component itself? And we should not mix px and rem